GIS.cpp

```diff
@@ -76,3 +74,35 @@ std::pair<Coordinates, EntityId> GIS::getWayClosestPoint(const Coordinates &) {
     std::pair<Coordinates, EntityId> p(coord, "something");
     return p;
 }
+
+
+std::vector<EntityId> GIS::loadEntities(rapidjson::Document &document) {
+    std::vector<EntityId> entityIds;
+    for (auto &jsonEntity : document.GetArray()) {
+        try {
+            std::unique_ptr<Entity> entity = entityJsonParser->parse(jsonEntity, *this);
+            EntityId entityId = entity->getId();
+            // if entityId not loaded yet
+            if (entities.find(entityId) == entities.end()) {
+                entities.emplace(entityId, std::move(entity));
+                entityIds.push_back(entityId);
+//                TODO insert entity to grid
+            } else {
+//                TODO print to log that id is not unique
```

Comment:
I think for now we can throw a runtime error, and later we can just add a catch for it

Reply:
But this is the function to handle the errors. Where can we handle errors if not here?